docs/source/membership/membership.md

```diff
@@ -1,43 +1,86 @@
+network. MSP's are also used to define the organizations on a network that
+are recognized by other organizations.
+
+Finally, consider if you want to join an _existing_ network,
+you need a way to turn your identity into something that is recognized
+by the network. The MSP is the mechanism that enables you to participate on
+a permissioned blockchain network. To transact on a Fabric network you
+need to:
+
+- Have an identity issued by a CA that is trusted by the network.
+- Become a member of an _organization_ that is recognized and approved
+by the network members. The MSP is how the identity is linked to the
+membership of an organization. Membership is achieved by adding your
+public key (also known as certificate, signing cert, or signcert) to
+your organization’s MSP.
+
+## What is an MSP?
+
+Despite its name, the Membership Service Provider does not actually provide
+anything. Rather, the implementation of the MSP requirement is a folder
+that contains a list of related identities and is used to store users's
+certificates or in the case of an organization or channel, related identities.
+Whereas Certificate Authorities generate the certificates that represent
+identities, the MSP contains a list of permissioned identities.
+
+The MSP identifies which Root CAs and Intermediate CAs are accepted to define
+the members of a trust domain, e.g. a node, organization, or channel, either by listing the
 identities of their members, or by identifying which CAs are authorized to
 issue valid identities for their members, or --- as will usually be the case ---
 through a combination of both.
 
 The power of an MSP goes beyond simply listing who is a network participant or
-member of a channel. An MSP can identify specific **roles** an actor might play either
-within the scope of the organization the MSP represents (e.g., admins, or as members
-of a sub-organization group), and sets the basis for defining **access privileges**
-in the context of a network and channel (e.g., channel admins, readers, writers).
-
-The configuration of an MSP is advertised to all the channels where members of
-the corresponding organization participate (in the form of a **channel MSP**). In
-addition to the channel MSP, peers, orderers, and clients also maintain a **local MSP**
-to authenticate member messages outside the context of a channel and to define the
-permissions over a particular component (who has the ability to install chaincode on
-a peer, for example).
+member of a channel. It is the MSP that turns an identity into a **role** by
+identifying specific roles an actor might play within the scope of the
+organization the MSP represents (e.g., admins, or as members of a
+sub-organization group), and sets the basis for defining **access
+privileges** in the context of a network and channel e.g., channel admins,
+readers, writers). Also, when a user is registered with a CA, a role of
+admin, peer, client, orderer, or member must be associated with the
+user.
```

Comment:
The problem I've discovered is that before you can understand the folder structure you need to first understand Local and Channel MSPs and Organizations and OUs. So that's why the folder structure is last -- unfortunately.